client/src/components/App.jsx

```diff
@@ -1,14 +1,47 @@
 import React from 'react';
+import Details from './Details.jsx';
 
 class App extends React.Component {
-  constructor() {
-    super();
-    this.state = {};
+  constructor(props) {
+    super(props);
+    this.state = {
+      hostName: '',
+      city: '',
+      country: '',
+      gettingAround: '',
+      description: '',
+      guidebookId: null
+    };
   }
 
+  componentDidMount() {
+
+    // $.ajax({
+    //   url: '/api/rooms/11111111/neighborhood',
+    //   dataType: 'json',
+    //   method: 'GET',
+    //   contentType: 'application/json',
+    //   // data: JSON.stringify({jsonData: data}),
+    //   success: function(data) {
+    //     console.log(data);
+    //   },
+    //   error: function(error) {
+    //     console.log(error);
+    //   },
+    //
+    // });
```

Comment:
If you're not using this code (indicated by the fact that it's commented out) we should remove it from what we're shipping to production.

Reply:
A get request will be used here, but I haven't figured out how I am going to configured the request on once componentDidMount runs on page load. The template for this AJAX request seems workable. Will most likely refactor in a separate branch.

Reply:
Please remove commented out code from production. Doesn't matter if you're going to use it in the future.